DependencyInjection/Configuration.php

```diff
@@ -38,10 +49,11 @@ public function getConfigTreeBuilder()
                             ->booleanNode('fullscreen')->defaultTrue()->end()
                             ->booleanNode('include_assets')->defaultTrue()->end()
                             ->scalarNode('tinymce_popup_path')->defaultValue('')->end()
-                            ->booleanNode('enableUserIntegration')->defaultFalse()->end()
+                            ->booleanNode('enable_user_integration')->defaultFalse()->end()
```

Comment:
Currently this node is not used, because it was aimed to provide some user permission based access.